src/pages/Strategies/components/StrategyPageHeader/StrategyPageHeader.style.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+
+import { theme } from '@/component-library';
+
+const StyledStrategyPageHeader = styled.div`
```

Comment:
You dont need most of these styles. You should be using `Flex` props that handle all these